packages/mock-app/src/components/GenericFeature/GenericFeature.tsx

```diff
@@ -141,7 +142,12 @@ export const GenericFeature: React.FC<IGenericFeatureProps> = ({ components, ser
             break;
           case ComponentType.Submit:
             props.onClick = async () => {
-              await executeServices(services, state);
+              try {
+                await executeServices(services, state);
+                toastMessage({ status: Status.success, message: 'Submit success' });
```

Comment:
I do not think we should put the `toastMessage` with `Submit` message here. I believe in the future, we will have more buttons with onClick props such as ScanQR or Download buttons. If we show to submit a message, it does not make sense.